authorizationService/handler.js

```diff
@@ -0,0 +1,39 @@
+module.exports.basicAuthorizer = function(event, context, callback) {
```

Comment:
Have you tested 401 and 403 errors in browser?
Do you have any problems with CORS?